src/context/chains.ts

```diff
@@ -1,14 +1,17 @@
 import { z } from 'zod';
 
 import { GithubRegistry, chainMetadata } from '@hyperlane-xyz/registry';
-import { ChainMap, ChainMetadata, ChainMetadataSchema } from '@hyperlane-xyz/sdk';
+import { ChainMap, ChainMetadata, ChainMetadataSchema, RpcUrlSchema } from '@hyperlane-xyz/sdk';
 
 import { chains as ChainsTS } from '../consts/chains.ts';
 import ChainsYaml from '../consts/chains.yaml';
 import { config } from '../consts/config.ts';
 import { cosmosDefaultChain } from '../features/chains/cosmosDefault';
+import { tryParseJson } from '../utils/json.ts';
 import { logger } from '../utils/logger';
 
+const RPC_OVERRIDES = process.env['NEXT_PUBLIC_RPC_OVERRIDES'];
```

Comment:
access this safely in src/consts/config like the rest of the env vars

Reply:
Nice I didn't see this file